bin/npm.cmd

```diff
@@ -1,17 +1,6 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-:: Try to find git.exe in path
```

Comment:
> Are there not scenarios where atom or a user might fire of this npm.cmd
> without going via apm.cmd?

As far as Atom is concerned I am pretty sure we never invoke `npm` directly. I think the same holds true for users, because they'll interface with `npm` transparently via `apm`. Maybe I am missing something, though? Did you have anything specific in mind, @damieng?